def.bzl

```diff
@@ -45,6 +45,21 @@ DEFAULT_LANGUAGES = [
     Label("//language/go:go_default_library"),
 ]
 
+def _valid_env_variable_name(name):
+    """ Returns if a string is in the regex [a-zA-Z][a-zA-Z0-9_]*
+
+    Given that bazel lacks support of regex, we need to implement
+    a poor man validation
+    """
+    if not len(name):
+        return False
+    for (index, character) in enumerate(name.elems()):
+        if (index == 0 and character.isalpha()) or \
```

Comment:
I don't think this correctly includes the case where the name starts with `_`. How about `character.isalpha() or (index != 0 and character == "_")`?

Also consider renaming character and index to c and I to shorten the line - for such a simple loop, long names don't add much.

Reply:
ok good catch then the regex is something like `[a-zA-Z_][a-zA-Z0-9_]*` let me rearrange that

Reply:
alright got it fixed and with a test!